### ISSUE TITLE
Swapping places of two pads

### ISSUE THREAD

**Issue by unfa** [openAVproductions/openAV-Fabla2#58]
One thing I really missed in Fabla, and also would love to see in Fabla 2 is rearranging the samples in the 4x4 grid.

The layout of the sounds is very important for organising stuff in a logica lmanner and also live performance might sometimes require a different layout to optimize for user performance on a MIDI controller.

Right now I don't see any easy way to swap places of two pads.

**Comment by harryhaaren** [comment]
Yes, this has been feature-requested by a few users, and is on the todo-list. i presume all layers should be swapped between the pads? AKA: multi-layered samples all move?

**Comment by unfa** [comment]
I'm not sure how the layers work yet, so I can't tell. Maybe it'd be good give users some control over this.
